### ISSUE TITLE
Version 6.0.0 build error on osx

### ISSUE THREAD

**Issue by RHavar** [barrysteyn/node-scrypt#115]
Trying: `npm install scrypt` I'm getting the error

```
> node-gyp rebuild

  SOLINK_MODULE(target) Release/copied_files.node
  CC(target) Release/obj.target/scrypt_wrapper/src/util/memlimit.o
  CC(target) Release/obj.target/scrypt_wrapper/src/scryptwrapper/keyderivation.o
../src/scryptwrapper/keyderivation.c:27:10: fatal error: 'sha256.h' file not found
#include "sha256.h"
         ^
1 error generated.
make: *** [Release/obj.target/scrypt_wrapper/src/scryptwrapper/keyderivation.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:100:13)
gyp ERR! stack     at ChildProcess.emit (events.js:185:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 15.0.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/rsmith/bench/test/node_modules/scrypt
gyp ERR! node -v v5.6.0
gyp ERR! node-gyp -v v3.2.1
gyp ERR! not ok 
```

**Comment by barrysteyn** [comment]
This seems very weird. The file is there, and the include path is there as well. The file is at `./scrypt/scrypt-1.2.0/libcperciva/alg/sha256.h`

Can you please make sure that the file is present on your pc. If it is there, then try change on line 27 in `src/scryptwrapper/keyderivation.c` `#include "sha256.h"` to `#include <sha256.h>`

**Comment by RHavar** [comment]
`scrypt/scrypt-1.2.0` doesn't seem to be included at all?

You can download the tar here: http://registry.npmjs.org/scrypt/-/scrypt-6.0.0.tgz

**Comment by barrysteyn** [comment]
What.  This is crazy - why is it not there. It most certainly is present on
Git.

Would you have any idea why?


Barry Steyn
Blog <http://doctrina.org> LinkedIn <http://ca.linkedin.com/in/barrysteyn>
GitHub <https://github.com/barrysteyn>

**Comment by barrysteyn** [comment]
Okay, stupid me. I figured out the problem. I have published version 6.01,
it should fix the problem

I am really sorry, and thanks for brining this up to my attention. Please
try it and tell me if it works.


Barry Steyn
Blog <http://doctrina.org> LinkedIn <http://ca.linkedin.com/in/barrysteyn>
GitHub <https://github.com/barrysteyn>

**Comment by RHavar** [comment]
Thanks! Works great
